pymc3/model.py

```diff
@@ -366,9 +367,17 @@ def __new__(cls, *args, **kwargs):
             instance._parent = cls.get_contexts()[-1]
         else:
             instance._parent = None
-        theano_config = kwargs.get('theano_config', None)
-        if theano_config is None or 'compute_test_value' not in theano_config:
-            theano_config = {'compute_test_value': 'raise'}
+        theano_config = kwargs.get('theano_config', {})
+        if 'compute_test_value' not in theano_config:
+            theano_config['compute_test_value'] = 'raise'
+        if theano_config['compute_test_value'] != theano.config.compute_test_value:
```

Comment:
Sorry, I see. But I still don't see why you would check it agains the current value.